lib/src/main/java/com/futurewei/alcor/common/constants/Ipv6AddressMode.java

```diff
@@ -0,0 +1,10 @@
+package com.futurewei.alcor.common.constants;
+
+public class Ipv6AddressMode {
```

Comment:
Should we use enum instead?

Reply:
I don't think so ~ because "Ipv6AddressMode" need String type value

Reply:
You could try something like https://howtodoinjava.com/java/enum/java-enum-string-example/